bin/beaver-install

```diff
@@ -5,6 +5,25 @@
 # http://www.boost.org/LICENSE_1_0.txt)
 set -eu
 
+while [ "$#" -ne 0 ] && [ "$1" != "--" ]
+do
+    case "$1" in
+        -t)
+        BEAVER_DOCKER_IMG="$2"
+        shift
+        shift
+        ;;
+        -d)
+        BEAVER_CONTEXT_DIR="$2"
+        shift
+        shift
+        ;;
+        *)
+        shift
+        ;;
+    esac
+done
+
```

Comment:
This looks hacky, why not using `getopt` instead?

Reply:
Also, it would be nice to add some help message and/or update the README.

Reply:
Is there any simple way to split on `--` with getopt?

Reply:
`getopt` will automatically stop there:

```

while getopts ":ht" opt; do
  case ${opt} in
    h )
        echo "Help"
        shift
      ;;
    t ) # process option t
        echo "T"
        shift
      ;;
    \? ) echo "Usage: cmd [-h] [-t]"
      ;;
  esac
done

echo "Remainder: $@"

```

```
nemanjaboric@labs-129:/home/nemanjaboric  git:(config-scripts*) $ ./script.sh -h -t -- nemanja
Help
T
Remainder: -- nemanja
nemanjaboric@labs-129:/home/nemanjaboric  git:(config-scripts*) $ ./script.sh -h -- -t nemanja
Help
Remainder: -- -t nemanja
```

Reply:
@nemanja-boric-sociomantic awesome, thanks!